x-pack/plugins/lens/common/expressions/datatable/sorting.tsx

```diff
@@ -7,8 +7,7 @@
 
 import versionCompare from 'compare-versions';
 import valid from 'semver/functions/valid';
-import ipaddr from 'ipaddr.js';
-import type { IPv4, IPv6 } from 'ipaddr.js';
+import ipaddr, { IPv4, IPv6 } from 'ipaddr.js';
```

Comment:
nit: I discovered the other day that it is possible also to mix type and non-type imports:

```suggestion
import ipaddr, { type IPv4, type IPv6 } from 'ipaddr.js';
```